charts/outbox-syncer/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: outbox-syncer
 description: A Helm chart for Kubernetes
 type: application
 version: v2025.6.30
-appVersion: v2025.6.30
+appVersion: v1.0.0
```

Comment:
This is the first version. It should be more like `v0.0.1` or `v0.1.0` not `v1.0.0`